### PR TITLE
static blog

### PR DESCRIPTION
**Pull request by DanielW1987**
the whole blog is static for now

### CODE REVIEW

File: webapp/src/main/resources/templates/frontend/blog/top-6-releases-2020.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                            <iframe src="https://open.spotify.com/embed/track/0BP93aviHComMr2mmRHxVf"
+                                    width="300"
+                                    height="300"
+                                    allowtransparency="true"
+                                    allow="encrypted-media">
+                            </iframe>
+                        </div>
+                        <div class="col-7">
+                            <p>
+                                The Scottish one-man Black'n'Roll project blasted it's way into my world last year with this amazing record,
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/frontend/blog/top-6-releases-2020.html:108]:
```suggestion
                                The Scottish one-man Black'n'Roll project blasted its way into my world last year with this amazing record,
```

File: webapp/src/main/resources/templates/frontend/blog/top-6-releases-2020.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                            <iframe src="https://open.spotify.com/embed/track/2F69AdkUI98pBot3uGKD6K"
+                                    width="300"
+                                    height="300"
+                                    allowtransparency="true"
+                                    allow="encrypted-media">
+                            </iframe>
+                        </div>
+                        <div class="col-7">
+                            <p>
+                                The second album of Danish one-man project hits exactly where atmospheric black metal should: the beautiful
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/frontend/blog/top-6-releases-2020.html:154]:
```suggestion
                                The second album of Danish one-man project Afsky hits exactly where atmospheric black metal should: the beautiful
```

File: webapp/src/main/resources/templates/frontend/blog/future-plans.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                    </p>
+
+                    <h2 class="h5 mt-4">Notifications</h2>
+                    <p>
+                        Currently the concept of the site is that you have to actively visit it and inform yourself when which band is
+                        releasing a new album. By restricting to your favorite bands, you are quickly up to date, but this manual effort is
+                        of course uncomfortable. Exactly for this reason there will soon be a notification system that will notify you
+                        proactively. For the time being this will be done via email and you can choose whether you want to receive a
+                        summary about recently released and upcoming album releases of your favorite band and/or additionally be notified
+                        on the day of announcement and/or release. In addition to e-email, channels such as Telegram or RSS feed are also
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/frontend/blog/future-plans.html:36]:
```suggestion
                        on the day of announcement and/or release. In addition to e-mail, channels such as Telegram or RSS feed are also
```

File: webapp/src/main/resources/templates/frontend/blog/future-plans.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+                        conceivable in the future.
+                    </p>
+
+                    <h2 class="h5 mt-4">Desktop first, mobile second</h2>
+                    <p>
+                        One or the other will have noticed it already, but mobile the Metal Detector is rather a disaster and not really
+                        usable. Of course this does not fit to today's and we want to change that as soon as possible, because we want to
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/frontend/blog/future-plans.html:43]: Does not fit in today's what? 😉

File: webapp/src/main/resources/templates/frontend/blog/future-plans.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+                        be available and usable for you on the go. A big construction site this year will therefore be the optimization of
+                        the website for mobile devices. Currently there are no plans to release the Metal Detector as a native app for
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/frontend/blog/future-plans.html:45]:
```suggestion
                        be available and usable for you on the go. A big construction site this year will therefore be the website's 
                        optimization for mobile devices. Currently there are no plans to release the Metal Detector as a native app for
```

File: webapp/src/main/resources/templates/frontend/blog/future-plans.html [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+                        Android and iPhone. We definitely lack the manpower for that at the moment.
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/frontend/blog/future-plans.html:46]:
```suggestion
                        Android and iPhone. We definitely lack the (wo)manpower for that at the moment.
```
For the fun of it? :)

**Review by hedgehog1833 — COMMENTED** [review]
Looks nice, just some textual improvements (also in my stuff 😅)

File: webapp/src/main/resources/templates/frontend/blog/future-plans.html (hunk 5 above)
Comment (hedgehog1833) [reply webapp/src/main/resources/templates/frontend/blog/future-plans.html:45]: Please check first if this suggestion does not kill the formatting.

File: webapp/src/main/resources/templates/frontend/blog/future-plans.html (hunk 6 above)
Comment (DanielW1987) [reply webapp/src/main/resources/templates/frontend/blog/future-plans.html:46]: only womanpower :dancing_women: !

**Review by hedgehog1833 — APPROVED** [review]

**Pull request merged**